src/mongo/Config.class.php

```diff
@@ -2110,6 +2110,14 @@ public static function getResqueServer()
         return $resqueServer;
     }
 
+    /**
+     * @return int|string
+     */
+    public static function getResqueDatabase()
+    {
+        return self::getenv(RESQUE_DATABASE, 0);
```

Comment:
I don't think we should warn about it missing because it is optional and has a default.